src/main/java/com/mcmoddev/mmdbot/commands/staff/CmdMute.java

```diff
@@ -64,8 +80,8 @@ protected void execute(CommandEvent event) {
             if (args.length > 2) {
                 TimeUnit unit1;
                 try {
-                    unit1 = TimeUnit.valueOf(args[2].toUpperCase());
-                } catch (IllegalArgumentException e) {
+                    unit1 = TimeUnit.valueOf(args[2].toUpperCase(Locale.ENGLISH));
```

Comment:
From my knowledge, usually `Locale.ROOT` is used for `toUpperCase` and `toLowerCase` calls (to be safe against the turkish character problem). While I believe `Locale.ENGLISH` works in that regard as well, it may be safer to just go with `Locale.ROOT`.

Reply:
Think that's all of them